components/Newsfeed/Newsfeed.tsx

```diff
@@ -0,0 +1,276 @@
+        onFilterChange={onFilterChange}
+        orgFilter={orgFilter}
+        billFilter={billFilter}
+      />
+    )
+  }
+
+  return (
+    <>
+      {loading ? (
+        <Row>
+          <Spinner animation="border" className="mx-auto" />
+        </Row>
+      ) : (
+        <>
+          {profile ? (
+            <>
+              <StyledContainer>
+                <Header>
+                  <HeaderTitle>Newsfeed</HeaderTitle>
+                  {isMobile ? (
+                    <Filters />
+                  ) : (
+                    <Col className="d-flex flex-column">
+                      <Filters />
+                    </Col>
+                  )}
+                </Header>
+                {noDisplay ? (
+                  <div className="pb-4">
+                    <AlertCard
+                      header={`No Results`}
+                      subheader={``}
+                      timestamp={``}
+                      headerImgSrc={``}
+                      bodyImgSrc={``}
+                      bodyImgAltTxt={``}
+                      bodyText={`There are no news updates for your current followed topics`}
+                    />
+                  </div>
+                ) : (
+                  <>
+                    {notificationsDisplay.map(
+                      (element: ElementProps, index: number) => (
+                        <div className="pb-4" key={index}>
+                          <AlertCard
+                            header={element.header}
+                            subheader={element.subheader}
+                            timestamp={element.timestamp}
+                            headerImgSrc={`${
+                              element.type === `org`
+                                ? `/profile-org-white.svg`
+                                : ``
+                            }`}
+                            bodyImgSrc={``}
+                            bodyImgAltTxt={``}
+                            bodyText={element.bodyText}
+                          />
+                        </div>
+                      )
+                    )}
+                  </>
+                )}
+                <SearchContainer>
+                  <Pagination className="d-flex justify-content-center mt-2 mb-3" />
+                </SearchContainer>
+              </StyledContainer>
+            </>
+          ) : (
+            <ErrorPage statusCode={404} withDarkMode={false} />
+          )}
+        </>
+      )}
+    </>
+  )
+}
+
+function FilterBoxs({
+  isMobile,
+  onFilterChange,
+  orgFilter,
+  billFilter
+}: {
+  isMobile: boolean
+  onFilterChange: any
+  orgFilter: boolean
+  billFilter: boolean
+}) {
+  return (
+    <>
+      <Row
+        className={`${
+          isMobile ? "justify-content-center" : "justify-content-end"
+        }`}
+      >
+        <div className="form-check checkbox">
+          <input
+            className="form-check-input"
+            type="checkbox"
+            value=""
```

Comment:
I think you can omit this if we're relying on the `checked` attribute

Reply:
done